### PR TITLE
docs: document production usage

### PR DESCRIPTION
**Pull request by domachine**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,13 @@ The project uses the [config](https://www.npmjs.com/package/config) npm package
 
 [(back to top)](#bsi-secvisogram-csaf-validator-service)
 
+## Using in production
+
+1. Install production dependencies using `npm ci --omit=dev` from within the `backend` folder
+2. Transfer the `backend` folder to your server or image
+3. Make sure to set the environment variable `NODE_ENV` to `production
```
Comment (valentindey) [inline README.md:45]:
```suggestion
3. Make sure to set the environment variable `NODE_ENV` to `production`
```

**Review by valentindey — CHANGES_REQUESTED** [review]

File: backend/config/default.json [hunk 2]
```diff
@@ -1,5 +1,4 @@
 {
   "ip": "127.0.0.1",
-  "port": 8082,
-  "csafValidatorLibMock": false
```
Comment (tschmidtb51) [inline backend/config/default.json:4]: jooc: Why do we remove the `"csafValidatorLibMock": false`?

File: backend/config/development.json [hunk 3]
```diff
@@ -1,5 +1,4 @@
 {
   "ip": "0.0.0.0",
-  "port": 8082,
-  "csafValidatorLibMock": true
```
Comment (tschmidtb51) [inline backend/config/development.json:4]: Same question here (if another explanation applies).

File: README.md [hunk 4]
```diff
@@ -38,6 +38,13 @@ The project uses the [config](https://www.npmjs.com/package/config) npm package
 
 [(back to top)](#bsi-secvisogram-csaf-validator-service)
 
+## Using in production
+
+1. Install production dependencies using `npm ci --omit=dev` from within the `backend` folder
```
Comment (tschmidtb51) [inline README.md:43]: I suggest to add script in the `package.json` for that...

File: README.md [hunk 5]
```diff
@@ -38,6 +38,13 @@ The project uses the [config](https://www.npmjs.com/package/config) npm package
...
+2. Transfer the `backend` folder to your server or image
+3. Make sure to set the environment variable `NODE_ENV` to `production
+4. Configure the service using a `local-production.json` file in `backend/config`. All available parameters are outlined in `backend/config/development.json`. See [https://www.npmjs.com/package/config](https://www.npmjs.com/package/config) for more information on how to configure using different techniques such as environment variables.
```
Comment (tschmidtb51) [inline README.md:46]: Shouldn't the `default.json` be referenced here? Do we recommend the use a reverse proxy (e.g. "The use of a reverse proxy is recommended.")? (If not, that's also okay - just checking; if so, why?)

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]
In general, ok-ish. Some detail questions (just out of curiosity and where a step might be missing) and suggestions.

File: backend/config/default.json (hunk 2 above)
Comment (domachine) [reply backend/config/default.json:4]: This option is not needed. It was a noop.

File: README.md (hunk 4 above)
Comment (cloeser) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (cloeser) [reply README.md]: The usage of a separate configuration should be the preferred way. I added the recommendation of a reverse proxy.

**Review by valentindey — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -23,8 +23,20 @@ This is a service to validate documents against the [CSAF standard](https://docs
...
+  folder by running `npm run dist`
+- copy the content of the dist folder to your working directory
+- Make sure to set the environment variable `NODE_ENV` to `production`
+- Configure the service using a `local-production.json` file in
+  `backend/config`. All available parameters are outlined in `backend/config/development.json`. See [https://www.npmjs.com/package/config](https://www.npmjs.com/package/config) for more information on how to configure using different techniques such as environment variables.
+- start the service with `node backend/server.js`
+
+To manage the process you can use Docker or an init system of your choice.
+
+You most likely also want to run this behind a reverse proxy to handle SSL
```
Comment (tschmidtb51) [inline README.md:36]: Just a minor here:
```suggestion
You most likely also want to run this behind a reverse proxy to handle TLS

**Review by tschmidtb51 — DISMISSED** [review]
In general, I approve. Please add the small change as noted before merging.

**Review by cloeser — APPROVED** [review]

**Pull request merged**